.gitignore

```diff
@@ -4,3 +4,4 @@ schedule_lessons/db.sqlite3
 schedule_lessons/staticfiles
 schedule_lessons/media/profile_pics/*
 !schedule_lessons/media/profile_pics/.gitkeep
+.DS_Store
```

Comment:
do we need this here? seems like it accidentally got added due to the way macs work

Reply:
I added it there so it doesn't keep pushing that useless file in the repo